test/e2e/s3-read.test.ts

```diff
@@ -93,6 +96,8 @@ describe('S3 Resource Selector - Read mode', () => {
     'shows alert about missing versions',
     setupTest(async page => {
       await page.setUriInputValue('s3://bucket-officia/particle-1ns.sim');
+      // click somewhere to blur the input and trigger validation
+      await page.click(createWrapper().findHeader().toSelector());
```

Comment:
Now that we're copying the same comment and the statement it in multiple places, how about just making a function out of it?

```tsx
async function waitForValidation() {
      // click somewhere to blur the input and trigger validation
      await page.click(createWrapper().findHeader().toSelector());
}
```

Reply:
That click to the header is only needed to trigger a blue on the input for the validation to appear. It can as well be a click on another element or pressing Tab. The comment is needed to explain that it is not the click itself that matters but the blur. I am hesitating to create a function for it because: 1) it is hard to come up with a clear enough name, the "wait for validation" is not correct; 2) it makes an assumption that the header exists which might not be true for some new page if we want to reuse the function in there.

Reply:
If we can come up with a proper name for the function, the name of the function could be very good replacement for the comment that we have repeated several times. That being said, if it's not applicable in this context then let's go with the comment 👍